templates/TagCloud.html.twig

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html.twig' %}
+{% block content %}
```

Comment:
Remove extends and block twig commands

Reply:
Not needed, Tag Cloud shouldn't be placed on every page according to the initial task.